distributed/scheduler.py

```diff
@@ -2057,35 +2058,34 @@ def transition_processing_memory(
                     ws,
                     key,
                 )
-                return {}, worker_msgs, client_msgs
+                return recommendations, worker_msgs, client_msgs
 
+            has_compute_startstop: bool = False
+            compute_start: double
+            compute_stop: double
             if startstops:
-                L = list()
+                startstop: dict
```

Comment:
I thought we did more with `L` but I don't see it being used anywhere except for recording `compute_start, compute_stop`

Reply:
Yeah exactly that's what I found as well, which is why I rewrote this to avoid creating an explicit `list`. Plus we only seem to use the first element of that `list`. So we just capture that in the `for`-loop below